lib/Sabre/Place/PlaceRoot.php

```diff
@@ -83,7 +83,9 @@ public function createDirectory($name) {
 	}
 
 	/**
-	 * @return PlacePhoto[]
+	 * @return (PlaceFile|PlacePhoto)[]
```

Comment:
I think it only returns `PlacePhoto`.
Those names are really confusing :see_no_evil:

Reply:
If psalm found the two, then something is wrong :wink:

Reply:
Ah, it is because of the declaration of `$children` !
Fixed :)